File: python/estrutura_condicional/jogo.py
inicial = int(input("Hora inicial: "))
final = int(input("Hora final: "))

if inicial < final:
    duracao = final - inicial
else: 
    duracao = 24 - (inicial - final)
    
print(f"O JOGO DUROU {duracao} HORA(S)")

File: c/estrutura_condicional/jogo.c
#include <stdio.h>

int main(void)
{
    int inicial, final;

    printf("Hora inicial: ");
    scanf("%d", &inicial);

    printf("Hora final: ");
    scanf("%d", &final);

    if (inicial < final) 
    {
        printf("O JOGO DUROU %d HORA(S)\n", final - inicial);
    }
    else {
		printf("O JOGO DUROU %d HORA(S)\n", 24 - (inicial - final));
    }
}
